kilkari/src/main/java/org/motechproject/nms/kilkari/service/impl/MctsBeneficiaryImportServiceImpl.java

```diff
@@ -190,6 +195,11 @@ public boolean importMotherRecord(Map<String, Object> record) {
             return false;
         }
 
+        //validate if an ACTIVE child is already present for the mother. If yes, ignore the update
+        if (childAlreadyPresent(mother.getBeneficiaryId())) {
+            return false;
```

Comment:
I would think that you would want to log to subscription error here, right? That way, in the future (after child is done) when bbc comes back and asks why a mother wasn't created.. you have an audit trail.